lib/exception_hunter/middleware/sidekiq_hunter.rb

```diff
@@ -0,0 +1,58 @@
+module ExceptionHunter
+  module Middleware
+    # Middleware to report errors
+    # when a Sidekiq worker fails
+    #
+    class SidekiqHunter
+      TRACK_AT_RETRY = [0, 3, 6, 10].freeze
+      JOB_TRACKED_DATA = %w[
+        queue
+        retry_count
+      ].freeze
+      ARGS_TRACKED_DATA = %w[
+        job_class
+        job_id
+        arguments
+        enqueued_at
+      ].freeze
+
+      def call(_worker, context, _queue)
+        yield
+      rescue Exception => exception # rubocop:disable Lint/RescueException
```

Comment:
2old school

Reply:
👴 I like to disable linters close to the code so people can fix them if they see something I did not. Plus moving code around breaks YML rules